src/core/components/link/ButtonLink.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { SvgExternal } from '@guardian/src-icons';
+import { ThemeProvider } from '@emotion/react';
+import { ButtonLink } from './ButtonLink';
+import { linkDefault } from './index';
+import { ButtonLinkProps } from './ButtonLink';
+import type { Story } from '../../../@types/storybook-emotion-10-fixes';
+import { asPlayground, asChromaticStory } from '../../../lib/story-intents';
+
+export default {
+	title: 'Source/src-link/ButtonLink',
+	component: ButtonLink,
+	args: {
+		priority: 'primary',
+		subdued: false,
+		icon: <SvgExternal />,
+		iconSide: 'left',
+	},
+	argTypes: {
+		icon: {
+			options: ['undefined', 'SvgExternal'],
+			mapping: {
+				undefined: undefined,
+				SvgExternal: <SvgExternal />,
+			},
+			control: { type: 'radio' },
+		},
+	},
+};
+
+const Template: Story = (args: ButtonLinkProps) => (
+	<ButtonLink {...args}>Click Me</ButtonLink>
+);
+
+// *****************************************************************************
+
+export const Playground = Template.bind({});
+asPlayground(Playground);
+
+// *****************************************************************************
+
+export const PrimaryButtonLinkLightTheme = (args: ButtonLinkProps) => (
```

Comment:
we should align on whether `Light` is the same as `Default`. Right now I couldn't say i really know, which i'm guessing means other people might be in the same situation?

Reply:
Good point. They do seem to be used interchangeably at the moment which can be a bit confusing. @SiAdcock any thoughts on which we should standardise on?

Reply:
(not a blocker for this though – we'll need to align all of one set the other way)